lib/matplotlib/axes/_axes.py

```diff
@@ -6510,31 +6510,33 @@ def hist2d(self, x, y, bins=10, range=None, normed=False, weights=None,
         Other Parameters
         ----------------
         cmap : Colormap or str, optional
-            A :class:`matplotlib.colors.Colormap` instance.  If not set, use rc
+            A :class:`.colors.Colormap` instance.  If not set, use rc
```

Comment:
Since we are using 'obj' as the default role, `:class:` should now be left out for improved readability.